package.json

```diff
@@ -89,9 +89,10 @@
     ]
   },
   "devDependencies": {
-    "@storybook/addon-actions": "^4.1.1",
-    "@storybook/addon-links": "^4.1.1",
-    "@storybook/react-native": "^4.1.18",
+    "@storybook/addon-actions": "^5.1.3",
+    "@storybook/addon-links": "^5.1.3",
+    "@storybook/react-native": "^5.1.3",
+    "acorn": "^6.1.1",
```

Comment:
Upgrading Storybook broke the linter:

```
> eslint . --ext .js,.jsx

module.js:550
    throw err;
    ^

Error: Cannot find module 'acorn'
    at Function.Module._resolveFilename (module.js:548:15)
```

Adding `acorn` as a dep fixes it.

Reply:
Are there any docs that describe this?

Reply:
Not that I could find, although I'll continue looking. However I did notice that we were getting a peer dependency warning when running `npm install` that `acorn` was listed but we hadn't installed it, so I'm wondering if this was only working before due to a side effect.

Reply:
https://npm.community/t/failed-to-install-eslint-in-a-specific-situation/2975 maybe this